docs/DeveloperGuide.md

```diff
@@ -302,32 +302,126 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+**Use case 1: View List of Customers**
 
-**Use case: Delete a person**
+* **System**: SellSavvy
+* **Actor**: USer
+* **Use Case**: UC01 - View List of Customers
 
 **MSS**
 
```

Comment:
As per course website use case topics, perhaps removing the whitespace after MSS and following subsections of use cases may be better?